plugins/woocommerce-blocks/assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -60,6 +61,7 @@ const productGallery = {
 		openDialog: () => {
 			const context = getContext();
 			context.isDialogOpen = true;
+			document.body.classList.add( 'modal-open' );
```

Comment:
Missed one.

Reply:
Nice catch! I just fixed it in https://github.com/woocommerce/woocommerce/pull/43378/commits/94bca53f19d2587dc93621842133d2e29380abf4